contrib_bots/lib/poll.py

```diff
@@ -0,0 +1,105 @@
+
+
+class PollHandler(object):
+    '''
+    This plugin facilitates polling.
```

Comment:
I'd think `This plugin allows users to create polls.` is a bit more friendly description :)